src/test/java/org/broadinstitute/barclay/argparser/CollectionArgumentUnitTests.java

```diff
@@ -303,6 +303,29 @@ public void testCollectionFromExpansionFile(
         Assert.assertEquals(argumentContainer.getList1(), expectedResult);
     }
 
+    class PositionalCollection {
```

Comment:
Probably want to make this class `private`.

Reply:
Done.